### PR TITLE
Added cloning from templete

### LINKED ISSUE #78: [Docs] New Page: "Getting Started - Creating a Design from Template"
#### Current State
A new docs page is needed to describe this process to users, who clone a design from a template in the Meshery Catalog.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

This PR fixes #78 

Added a new docs page to describe, cloning a design from a template in the Meshery Catalog. 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: content/en/cloud/getting-started/creating-a-design-from-templete.md.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description: With this guide, you can create amazing cloud native designs for your infrastructure from already avaliable templetes in the meshery catalog. 
+weight: 2
+---
+
+[Meshery Catalog](https://meshery.layer5.io/catalog) functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure. With Meshery Catalog, you can easily find,clone and deploy Kubernetes-based infrastructure and tools, making it easy to streamline your cloud native development and operations processes.
+
+##  Cloning a design from a template in the Meshery Catalog
+
+1. Sign into [Layer5 Cloud](https://meshery.layer5.io)
+2. Visit catalogs which will find inside the menu.
```
Comment (leecalcote) [inline content/en/cloud/getting-started/creating-a-design-from-templete.md.md:12]: Rephrase

File: content/en/cloud/getting-started/creating-a-design-from-templete.md.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+3. Search by technology or type and find the best templete for your infrastructure. You will already see some of the information just by hovering on it.
+4. For detailed information just click on it.
+5. Simply click the green clone button.
```
Comment (leecalcote) [inline content/en/cloud/getting-started/creating-a-design-from-templete.md.md:15]: Can’t users Clone from inside of MeshMap, too?

Comment (ptdatta) [reply content/en/cloud/getting-started/creating-a-design-from-templete.md.md:15]: Yes, but it's inside Meshmap but we are viewing this doc is inside the cloud that's why I didn't add it. Ok, I'll add it here too.

File: content/en/cloud/getting-started/creating-a-design-from-templete.md.md [hunk 3]
```diff
@@ -9,11 +9,11 @@ weight: 2
...
 
 1. Sign into [Layer5 Cloud](https://meshery.layer5.io)
-2. Visit catalogs which will find inside the menu.
+2. Explore the catalogs located in the menu section.
 3. Search by technology or type and find the best templete for your infrastructure. You will already see some of the information just by hovering on it.
 4. For detailed information just click on it.
 5. Simply click the green clone button.
 6. Now you have it. Navigate to [Meshery Playground](https://playground.meshery.io/)
-7. Go to the meshmap and you will see your cloned template here. If you con't see it just reload the page.
+7. Go to the Meshmap and you will see your cloned template here. If you con't see it just reload the page.
```
Comment (leecalcote) [inline content/en/cloud/getting-started/creating-a-design-from-templete.md.md:17]: Again, it's MeshMap 😬

File: content/en/cloud/getting-started/creating-a-design-from-templete.md.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+##  Cloning a design from a template in the Meshery Catalog
+
+1. Sign into [Layer5 Cloud](https://meshery.layer5.io)
+2. Explore the catalogs located in the menu section.
+3. Search by technology or type and find the best templete for your infrastructure. You will already see some of the information just by hovering on it.
+4. For detailed information just click on it.
+5. Simply click the green clone button.
+6. Now you have it. Navigate to [Meshery Playground](https://playground.meshery.io/)
+7. Go to the Meshmap and you will see your cloned template here. If you con't see it just reload the page.
```
Comment (leecalcote) [inline content/en/cloud/getting-started/creating-a-design-from-templete.md.md:17]:
```suggestion
7. Go to the MeshMap and you will see your cloned template here. If you con't see it just reload the page.
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**